ui/components/app/transaction-decoding/components/decoding/address/address.component.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
```

Comment:
This component controls the display and behaviour of each icon+address in the decoded transaction information:
![Screenshot from 2021-12-01 09-41-43](https://user-images.githubusercontent.com/7499938/144240456-ab7d1b34-0f41-4550-9838-4af261273b1f.png)